benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json

```diff
@@ -59,5 +60,6 @@
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "translationWorkOutstanding", "Label": "Translation outstanding", "DisplayOrder": 14},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 15},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 16},
-  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17}
-]
+  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17},
+  {"LiveFrom": "10/10/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-superuser", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting officer attending", "DisplayOrder": 18}
```

Comment:
If this is new we should add the current date in the LiveFrom such as 25/05/2022

Reply:
Shouldn't matter though right? LiveFrom dates are deprecated by CCD anyway and aren't actually used for anything?

Reply:
It doesn't actually effect anything but gives good insight on when we added stuff to trace it back if needed.